components/eoy_artist_list/_artist_item.jade

```diff
@@ -1,6 +1,6 @@
 .eoy-artist-item
   .eoy-artist-item__thumbnail
-    a(href="/artist/#{artist.slug}"): img(src="#{sd.API_URL}/api/v1/artist/#{artist.slug}/image")
+    a(href="/artist/#{artist.slug}"): img(src="/search/image/artist/#{artist.slug}")
```

Comment:
Ah perfect, I had started looking at if we can pass the xapp token as a GET param, this is much simpler 👍